### PR TITLE
Expose Subscription's billing date

### PR DESCRIPTION
**Pull request by RayeN**
We need this for SiteWit audit.

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by ekoval** [comment]
No tests required?

**Comment by RayeN** [comment]
Integration test already covers that SiteWit response is parsed without errors.

**Comment by RayeN** [comment]
But yeah, I can add a separate test for Subscription model

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by RayeN** [comment]
Done.

File: tests/campaigns/test_models.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from datetime import date
+from decimal import Decimal
+from unittest import TestCase
+
+
```
Comment (ekoval) [inline tests/campaigns/test_models.py]: Why two blank lines?

Comment (RayeN) [inline tests/campaigns/test_models.py]: My bad, fixing...

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by ekoval** [comment]
:+1:

**Pull request merged**